Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [".", "xtask"]
 [features]
 default = ["pulseaudio"]
 pulseaudio = ["libpulse-binding"]
+pipewire = ["dep:pipewire"]
```

Comment:
I assume we don't want to make pipewire default since it's not as widespread as pulse,right?

Reply:
Yeah, I think so.